eagleEvents/run_ga.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+from eagleEvents.seating_chart_ga import SeatingChartGA
+from eagleEvents.models import Event
+
+def run():
+    ga = SeatingChartGA(Event.query.first())
```

Comment:
Can you query for a specific event?  First can come back different depending on the database

Reply:
Done.